presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -361,6 +363,35 @@ public void testCorrelatedSubqueries()
                 MorePredicates.<PlanOptimizer>isInstanceOfAny(AddLocalExchanges.class, CheckSubqueryNodesAreRewritten.class).negate());
     }
 
+    @Test
+    public void testStreamingAggregationForCorrelatedSubquery()
+    {
+        assertPlanWithSession("SELECT name, (SELECT max(name) FROM region WHERE regionkey = nation.regionkey AND length(name) > length(nation.name)) FROM nation",
```

Comment:
please put each argument in separate line like:

```
method(
    arg1,
    arg2,
    ...,
    lastArg);
```

Reply:
@kokosing I believe I'm doing this except in cases where it consumes too much vertical space without adding any value. I think this is consistent with the rest of the code in this file. Is there a specific change you'd like me to make? Could you point it out?